packages/aikit-ui/src/blocks/Chat.tsx

```diff
@@ -0,0 +1,272 @@
+"use client";
+
+import { createOpenAI } from "@ai-sdk/openai";
+import { generateText, type UIMessage } from "ai";
+import { Bot, PaperclipIcon, IterationCcw } from "lucide-react";
+import { useEffect, useState } from "react";
+
+import {
+  Conversation,
+  ConversationContent,
+  ConversationScrollButton,
+} from "../_ai-elements/conversation";
+import { Message as Message, MessageContent } from "../_ai-elements/message";
+import {
+  PromptInput,
+  PromptInputButton,
+  PromptInputModelSelect,
+  PromptInputModelSelectContent,
+  PromptInputModelSelectItem,
+  PromptInputModelSelectTrigger,
+  PromptInputModelSelectValue,
+  PromptInputSubmit,
+  PromptInputTextarea,
+  PromptInputToolbar,
+  PromptInputTools,
+} from "../_ai-elements/prompt-input";
+import { Button } from "../_shadcn/ui/button";
+
+// Types based on models.schema.json
+type ModelsConfig = {
+  models: Array<{
+    alias: string;
+    type: string;
+    endpoint?: {
+      baseUrl: string;
+      fetch?: (
+        input: RequestInfo | URL,
+        init?: RequestInit,
+      ) => Promise<Response>;
+    };
+  }>;
+};
+
+export function Chat({ modelsConfig }: { modelsConfig: ModelsConfig }) {
+  const [currentModelAlias, setCurrentModelAlias] = useState(
+    modelsConfig.models.length > 0 ? modelsConfig.models[0].alias : "",
+  );
+  const [messages, setMessages] = useState<UIMessage[]>([]);
+  const [input, setInput] = useState("");
+  const [isLoading, setIsLoading] = useState(false);
+
+  // Get current model config - only return a model if we have a valid alias and it exists
+  const currentModel = currentModelAlias
+    ? modelsConfig.models.find((m) => m.alias === currentModelAlias)
+    : undefined;
+
+  // Create OpenAI client based on current model (only if model exists)
+  const openai = currentModel
+    ? createOpenAI({
+        apiKey: "",
+        baseURL: currentModel.endpoint?.baseUrl || "",
+        fetch:
+          currentModel.endpoint?.fetch ||
+          ((input: RequestInfo | URL, init?: RequestInit) =>
+            fetch(input, {
+              ...init,
+              credentials: "include",
+            })),
+      })
+    : undefined;
+
+  useEffect(() => {
+    const handleGlobalKeyDown = (e: KeyboardEvent) => {
+      // Skip if already in an input
```

Comment:
Why? I could be in a totally different input and want to jump to the playground chat.